### ISSUE TITLE
[Accessibility] Make `editor.action.toggleTabFocusMode` configurable in user settings

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#173451]
CC @meganrogge 

Currently, the state of `editor.action.toggleTabFocusMode` is not preserved and users need to toggle this on and off manually each time they reload VSCode.

Two discussion points are:

1. What if users want to preserve this state?

2. What if users want to have different tab key behaviors between editor and terminal? For example, I prefer to toggle off tab move focus in editor whereas I would like to keep it on in terminal.

## Expected Behaviors

It would be great if users could configure the tab key behavior in editor and terminal context separately.

## System Info

Version: 1.76.0-insider (system setup)
Commit: c4a609f94329b17d91cd13a4fd113d1abfe5ec1f
Date: 2023-02-03T05:23:17.202Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by Tyriar** [comment]
A downside of having this as a setting is your settings can change a lot. Maybe the best of both worlds is something like the command toggles it without changing the setting, and the setting defines the initial state when the editor loads?

**Comment by meganrogge** [comment]
> A downside of having this as a setting is your settings can change a lot.

What do you mean by this? @Tyriar 

I believe the toggling action was likely introduced to account for operating in different contexts IE the terminal vs editor. It makes sense that a user would always want it on/off for each context and not want to have to toggle it.

**Comment by Tyriar** [comment]
I believe it's a state that a user would typically toggle on and off frequently, settings.json is meant to be a more stable and not change so much.

**Comment by meganrogge** [comment]
Yep, let's keep the command to toggle and add separate settings for the terminal and workbench so it persists on reload and doesn't require toggling between the different components

**Comment by meganrogge** [comment]
I think this action should be available via the command palette

**Comment by meganrogge** [comment]
this is very involved if we want the status bar item to reflect the persisted state/ setting value as the linked commit shows

**Comment by meganrogge** [comment]
Found a simpler approach
